hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotUtils.java

```diff
@@ -155,4 +162,30 @@ public static SnapshotInfo getNextActiveSnapshot(SnapshotInfo snapInfo,
     }
     return null;
   }
+
+  /**
+   * Return a map column family to prefix for the keys in the table for
+   * the given volume and bucket.
+   * Column families, map is returned for, are keyTable, dirTable and fileTable.
+   */
+  public static Map<String, String> getColumnFamilyToPrefixMap(
```

Comment:
Can we use getOzoneKey and getOzoneDirKey functions instead of rewriting it just like how other functions do it. https://github.com/apache/ozone/blob/245b2ca3cf6c9c33cc9658d6dc4ee2ed15d58fcd/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java#L828

Reply:
I looked into all the functions `getOzoneKey()`, `getOzoneDirKey()`, `getBucketKey()`, `getOzonePathKey()` and `getOzoneDeletePathKey()` in `OmMetadataManager`  but none of them is as per our requirement. Either they don't have tailing `/` or they don't use volumeId and bucketId.

For the sake of this PR, I created require helper function to SnapshotUtils and used it everywhere in the snapshot code. As of now, it is only required by snapshot code path only. We can revisit and move new helper function to `OmMetadataManager` if needed later.

Reply:
That sounds good.